synapse/models/risk.py

```diff
@@ -81,6 +81,8 @@ def getModelDefs(self):
                 ('risk:mitigation', ('guid', {}), {
                     'doc': 'A mitigation for a specific risk:vuln.',
                 }),
+                ('risk:mitname', ('str', {'lower': True, 'onespace': True}), {
```

Comment:
this one too :)  We can make an easy eventual migration that makes a form out of `:name` and `:names` :+1: